api/v1alpha1/common_types.go

```diff
@@ -172,4 +172,30 @@ type MoverConfig struct {
 	MoverResources *corev1.ResourceRequirements `json:"moverResources,omitempty"`
 	// MoverAffinity allows specifying the PodAffinity that will be used by the data mover
 	MoverAffinity *corev1.Affinity `json:"moverAffinity,omitempty"`
+	// MoverVolumes are PVCs or Secrets that should additionally be mounted to the mover job pod.
+	// This should only be used by advanced users.
+	// +optional
+	MoverVolumes []MoverVolume `json:"moverVolumes,omitempty"`
+}
+
+type MoverVolume struct {
+	// Name to give the volume when mounting under /mnt in the mover job pod.
+	// For example if name is 'my-pvc' then this moverVolume will be mounted in the mover pod
+	// at /mnt/my-pvc
+	Name string `json:"name"`
```

Comment:
What about using a name like `mountPath` similar to the container spec? While this is different than container `mountPath`--- we're forcing it under `/mnt`, I think it's more descriptive than just `name`